static_src/stores/user_store.js

```diff
@@ -67,18 +68,27 @@ export class UserStore extends BaseStore {
         break;
       }
 
-      case userActionTypes.USER_INVITE_FETCH: {
+      case userActionTypes.USER_INVITE_TRIGGER: {
+        this._inviteDisabled = true;
         this._inviteError = null;
         this.emitChange();
         break;
       }
 
-      case userActionTypes.USER_ORG_ASSOCIATED: {
-        const user = Object.assign({}, { orgGuid: action.orgGuid }, action.user);
-        this._inviteInputActive = true;
+      case userActionTypes.USER_ASSOCIATED_ORG_DISPLAYED: {
+        let orgUser;
+        action.orgUsers.map(function(item){
```

Comment:
A potentially easier way to do this could be to use es6's `[].find`.

```js
const orgUser = action.orgUsers.find((orgUser) => orgUser.guid === action.userGuid);
```

That finds the user when it's guid is equal to the action user guid.